_topics/en/rendez-vous-routing.md

```diff
@@ -67,6 +67,12 @@ see_also:
   - title: Unannounced channels
     link: topic unannounced channels
 
```

Comment:
I think “Blinded Paths” may be the main name for this meanwhile, so perhaps the title and alias should be switched out?

Reply:
I agree blinded paths is now the main name for the thing we describe on this topic page.

For minor renames (e.g. "bech32" -> "bech32(m)"), I feel comfortable changing the YAML `title` field and creating an alias as the URL still makes sense (e.g. `/en/topics/bech32`).  For major renames (e.g. "SIGHASH_NOINPUT" -> "SIGHASH_ANYPREVOUT"), I want to also change the URL, which requires renaming the file, adding a redirect field, and potentially making other mildly invasive changes to the site.  I think that's best done in a separate PR, either an independent PR just for the rename or a PR which significantly updates the topic description.

This page is one of several that I plan to rename and significantly update in the near future, so I'm going to punt this until then.